system/jlib/jdebug.cpp

```diff
@@ -2333,6 +2503,11 @@ static class CMemoryUsageReporter: public Thread
                 tm.reset(NAMEDCOUNTPERIOD*1000);
             }
 #endif
+            if (traceMode&PerfMonUDP)
+            {
+                snmpStats.reportSnmpInfo();
+                udpStats.reportUdpInfo(0);
```

Comment:
Should we somehow grab udpTraceLevel setting value from config or roxie and pass in here ?